BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -578,7 +578,7 @@ def extract (self, name = '', file_path = ''):
 
             # create header entry
             auth_type_str = self.get_auth_type_str (self.header.auth_type)
-            key_file = 'TestSigningPrivateKey.pem' if auth_type_str.startswith('RSA') else ''
+            key_file = 'CONTAINER_KEY_ID' if auth_type_str.startswith('RSA') else ''
             alignment = self.header.alignment

```

Comment:
Use CONTAINER_KEY_ID and CONTAINER_COMP_KEY_ID while extracting component and creating layout.